src/esp/assets/ResourceManager.cpp

```diff
@@ -1099,10 +1102,12 @@ bool ResourceManager::loadSUNCGHouseFile(const AssetInfo& houseInfo,
                                          scene::SceneNode* parent,
                                          DrawableGroup* drawables) {
   ASSERT(parent != nullptr);
-  const std::string& houseFile = houseInfo.filepath;
+  std::string houseFile = Cr::Utility::Directory::join(
```

Comment:
Maybe only do this if the path doesn't exist?  Otherwise people always have to specify a relative path.

Reply:
It always exists ;) ... only it's relative in some cases (relative to CWD) and absolute in others -- in both cases `exists()` returns `true`.

And, (I think I mentioned this a few times already) `join()` works the same as Python's `os.path.join()` -- so if you `join(whatever, absolutePath)`, you *always* get `absolutePath` unchanged.

Reply:
Ah!  I didn't know python's `join` did that :)

Reply:
In my opinion that's *the* main feature of it that makes `join()` so immensely useful and failproof :)